src/main/java/net/io/kino/model/Ticket.java

```diff
@@ -19,33 +19,68 @@
     @JsonIdentityReference(alwaysAsId = true)
     private Seat seat;
 
-    private String cokolwiek;
+    @OneToOne
+    @JoinColumn
+    @JsonIdentityReference(alwaysAsId = true)
+    private Show show;
+
+    @OneToOne
+    @JoinColumn
+    @JsonIdentityReference(alwaysAsId = true)
+    private TicketType ticketType;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "order_id")
+    private Order order;
+
+    public Ticket() {
+    }
 
-    public Ticket() {}
 
```

Comment:
Nie wiem, czy ticket powinien mieć referencje do Order. Tworzy to niepotrzebną zależność rekurencyjną.

Reply:
https://en.wikibooks.org/wiki/Java_Persistence/ElementCollection  
Tutaj jest pokazane, jak zrobić aby Order miał listę Ticketów i było to mapowane na bazę danych.